docs/docsite/rst/network/user_guide/platform_index.rst

```diff
@@ -44,6 +45,12 @@ Settings by Platform
 +-------------------+-------------------------+-------------+---------+---------+----------+
 | Cisco IOS*        | ``ios``                 | ✓           |         |         | ✓        |
 +-------------------+-------------------------+-------------+---------+---------+----------+
+| Ruckus ICX*       | ``icx``                 | ✓           |         |         |          |
```

Comment:
I think this table is better sorted by full name ("Ruckus ICX"), rather than by network_os name (`icx`)